docker/travis_gcloud.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -ev
+if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
+	openssl aes-256-cbc -K $encrypted_2f660428f0db_key -iv $encrypted_2f660428f0db_iv -in istio-test.json.enc -out istio-test.json -d
```

Comment:
yes. as best I can tell, they were added when I used the travis CLI to encrypt the key file.